### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by katembu**
REINDEX and Update on Form Alias Indicators REQUIRED

### CODE REVIEW

**Comment by katembu** [comment]
@biyeun

**Comment by czue** [comment]
note for deployer - this is a reindex of the `mvp_indicators` database. I think it also might also require running `fab production reset_mvp_pillows` but @biyeun should confirm that. 

@katembu data will be behind while the tables are rebuilding. is that ok? I know we just finally fixed these reports

**Comment by katembu** [comment]
@czue If that the case let put it on hold till 10 July.

**Comment by czue** [comment]
ok - putting a don't merge tag on until later. @katembu probably good for you to ping again when you want this merged

**Comment by biyeun** [comment]
yep, this will require a reset of the pillows

File: custom/_legacy/mvp_indicators/mvp_chw_visits_indicators/views/all_visit_forms/map.js [hunk 1]
```diff
@@ -135,22 +132,30 @@ function(doc) {
...
-                        indicator_entries["child neonate"] = case_id;
-                    }
-                    if (age < 8*MS_IN_DAY) {
-                        indicator_entries["child 7days"] = case_id;
+                    if (isChildVisitForm(doc)) {
+                        indicator_entries['child under1'] = case_id;
+                        if (age < 180*MS_IN_DAY) {
+                            indicator_entries['child under6mo'] = case_id;
+                            if (indicators.exclusive_breastfeeding
+                                && indicators.exclusive_breastfeeding.value === 'yes') {
```
Comment (dimagimon) [inline custom/_legacy/mvp_indicators/mvp_chw_visits_indicators/views/all_visit_forms/map.js]: Bad line breaking before '&&'.

**Comment by katembu** [comment]
@czue this can be merged anytime from now.

**Comment by czue** [comment]
@biyeun can you review? note for deployer (is that @proteusvacuum?) that after this is merged will require a big preindex so recommend merging and kicking that off after today's deploy. also after this is deployed we will have to run:

`fab production reset_mvp_pillows`

**Comment by biyeun** [comment]
this looks good. we're still waiting till after deploy to merge, ya?

**Comment by proteusvacuum** [comment]
FYI, I reset the pillows after today's deploy.

**Comment by katembu** [comment]
@proteusvacuum is the reindexing complete?

**Comment by proteusvacuum** [comment]
@katembu the reindex finished, but looks like the pillows are still catching up. 
screen shot 2015-07-14 at 8 41 17 am

**Comment by katembu** [comment]
@proteusvacuum did the reindex complete 100%.  Am noticing some sites with less Indicator percentage than expected. Let me know so that I can raise issue if it not related.

**Comment by czue** [comment]
@katembu nope, still going. good thing we are switching over to UCR!

**Comment by katembu** [comment]
@czue @proteusvacuum  can you check for me if this completely finished. Reports for July are not correct.

**Comment by czue** [comment]
still running. cases are almost done. forms are still way behind.

**Comment by katembu** [comment]
Thanks. Would you mind sharing the percentage remaining?

**Comment by czue** [comment]
cases about 93%, forms about 56%

**Pull request merged**